.dockerignore

```diff
@@ -0,0 +1 @@
+.git/
```

Comment:
Should we add this to the template repo (and stactools itself)?

Reply:
Yeah, I think so. It's in my PR for stac-utils/stactools#136.
I'm going to look at the template repo tomorrow to see what else should be copied over.